modules/dynamic-plugins/con-dynamic-plugins-cache.adoc

```diff
@@ -49,11 +55,75 @@ spec:
                   claimName: dynamic-plugins-root
 ----
 
+[NOTE]
+To avoid adding a new volume, you must use the `$patch: replace` directive.
+
+=== Creating a PVC for Dynamic Plugins Cache using the Helm Chart
+For Helm Chart installations, if you require the dynamic plugin cache to persist across pod restarts, you must create a PersistentVolumeClaim (PVC) and configure the Helm Chart to use it.
+
+.Example creating the persistent volume definition
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-root
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+
+----
+
 [NOTE]
 ====
-Future versions of the {product-very-short} operator are planned to automatically create the PVC.
+This example uses `ReadWriteOnce` as the access mode which prevents multiple replicas from sharing the PVC across different nodes. 
+
+To run multiple replicas on different nodes, depending on your storage driver, you must use an access mode such as `ReadWriteMany`.
 ====
 
+To apply this PVC to your cluster, run the following command:
+[source,terminal]
+----
+oc apply -f pvc.yaml
```

Comment:
Similar point here. To be consistent, I think this should also be mentioned in the Operator section (or removed from here).